### DISCUSSION TITLE
Exit codes ?

### DISCUSSION THREAD

**Discussion by sundeepgoel72** [Breakthrough/DVR-Scan#96] [Q&A] [answered]
Hi, 

Does DVR-Scan return exit codes ? 

I want to use it in window batch file to delete a processed video file which does not have any motion event. Currently 0 is returned irrespective of motion found / extracted or not - which cannot be used for deciding further processing.

**Comment by Breakthrough** [comment, marked as answer]
DVR-Scan always returns 0 after successfully processing videos, and only returns non-zero when an error occurred.  This is typically what exit codes should be used for, not to convey information like number of detected events (if any).  I would suggest searching the command output for the text "No motion events detected in input."  On Windows, you can do this using the `findstr` command.

If you are so inclined, you could also modify the [`run_dvr_scan()` function in cli/controller.py](https://github.com/Breakthrough/DVR-Scan/blob/v1.5.1/dvr_scan/cli/controller.py#L180) for your particular use case.  Hope this helps!

**Reply by sundeepgoel72** [reply]
noted, thank you
